halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/config/artifacts/helm/HelmEditArtifactAccountCommand.java

```diff
@@ -39,14 +39,12 @@
       description = "Helm chart repository basic auth username"
   )
   private String username;
-  @Secret
```

Comment:
Do we need to remove these? I'm afraid this will break users work flows automating the CLI

Reply:
Those were initially put there in error, when I was still getting to know halyard. The annotation is supposed to be just on the config classes. So in this case, the @Secret is in HelmArtifactAccount. But we don't have to remove them if it will break things.

Reply:
sgtm